### PR TITLE
Mcc and ministry defaults

### PR DESCRIPTION
**Pull request by wrandall22**
This PR fixes some default values issues

### CODE REVIEW

**Comment by wrandall22** [comment]
@mattfred @frett

**Comment by frett** [comment]
So, I'm thinking loading current ministry & assignment would be a good place to utilize a ContentLoader to replace the SetCurrentMinistry AsyncTask. I'll probably dig in a bit tonight and setup the framework at least and try to help walk you through how they operate.

File: app/src/main/java/com/expidev/gcmapp/MainActivity.java [hunk 1]
```diff
@@ -341,6 +343,19 @@ private void getCurrentAssignment()
...
+    {
+        String chosenMinistry = preferences.getString("chosen_ministry", null);
+
+        if(chosenMinistry == null || currentMinistry == null || chosenMinistry.equals(currentMinistry.getName()))
+        {
+            return;
+        }
+
+        refreshAssignment = true;
+        refreshAssignment = !new SetCurrentMinistry().doInBackground(this);
```
Comment (frett) [inline app/src/main/java/com/expidev/gcmapp/MainActivity.java]: This would be run on the UI thread

Comment (wrandall22) [inline app/src/main/java/com/expidev/gcmapp/MainActivity.java]: So, even though it calls an async task method, it will be run in the UI thread?

Comment (frett) [inline app/src/main/java/com/expidev/gcmapp/MainActivity.java]: you are just calling it incorrectly. What you actually want to do for AsyncTasks is something like this: `new SetCurrentMinistry().execute(this);`

then sometime later the method `SetCurrentMinistry.onPostExecute(result)` will be triggered on the UI thread for you to handle what it is returning.

Comment (wrandall22) [inline app/src/main/java/com/expidev/gcmapp/MainActivity.java]: Gotcha

**Pull request merged**